src/cls/rgw/cls_rgw_ops.h

```diff
@@ -1002,56 +1002,80 @@ struct cls_rgw_lc_get_next_entry_op {
 WRITE_CLASS_ENCODER(cls_rgw_lc_get_next_entry_op)
 
 struct cls_rgw_lc_get_next_entry_ret {
-  pair<string, int> entry;
+  cls_rgw_lc_entry entry;
 
   cls_rgw_lc_get_next_entry_ret() {}
 
   void encode(bufferlist& bl) const {
-    ENCODE_START(1, 1, bl);
+    ENCODE_START(2, 2, bl);
```

Comment:
@yaozongyou let's keep backward compatible structure encoding. `ENCODE_START(2, 1, bl);`, and encode a `pair<string, int>(entry.bucket, entry.status)` as the first param.